charts/gardener-extension-provider-alicloud/values.yaml

```diff
@@ -69,6 +69,7 @@ config:
     storage:
       className: gardener.cloud-fast
       capacity: 25Gi
+  enableSshKeyRotate: false
```

Comment:
Why is this configurable in the provider config of the Alicloud extension? I mean, why would I want to disable this? The SSH rotation feature of Gardener is not optional but rather mandatory, so the provider extensions should always support it.